deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -358,10 +501,15 @@ services:
       GRAPH_HTTP_ADDR: 0.0.0.0:9120
       GRAPH_SPACES_WEBDAV_BASE: https://${OCIS_DOMAIN:-ocis.owncloud.test}
 
-      REVA_GATEWAY: storage-gateway:9142
+      GRAPH_LDAP_URI: ldaps://localhost:9235
+      GRAPH_LDAP_BIND_PASSWORD: ${IDM_SVC_PASSWORD:-changeme}
+
+      REVA_GATEWAY: gateway:9142
+
```

Comment:
the graph api shouldn't work without GRAPH_LDAP_INSECURE=true ? (see https://github.com/owncloud/ocis/issues/3818)